### PR TITLE
[RFR] Updates to cfme.tests.containers.test_manageiq_ansible_*

### PR DESCRIPTION
**Pull request by jawatts**
https://github.com/ManageIQ/integration_tests/issues/6484

{{ pytest: cfme/tests/containers/test_tags.py --use-provider cm-env2 --long-running }}

### CODE REVIEW

**Comment by shalomnaim1** [comment]
@jawatts according to the discussion we had with @psav, I think you need to add this test ```@pytest.mark.manual``` decoration and not remove them.

@psav please correct me if I am wrong.

File: cfme/tests/containers/test_tags.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        d = {}
+        ref = {'name': getattr(selected_entity, 'name', None),
+               'project_name': getattr(selected_entity, 'project_name', None),
+               'host': getattr(selected_entity, 'host', None),
+               'id': getattr(selected_entity, 'id', None),
+               'provider': provider}
+        for arg in ['name', 'project_name', 'host', 'id', 'provider']:
+            if arg in [att.name for att in klass.ENTITY.__attrs_attrs__]:
+                d.update({arg: ref[arg]})
+        return item_collection.instantiate(**d)
```
Comment (gshefer) [inline cfme/tests/containers/test_tags.py]: __REQUIRED__
You can change all the above lines to:
```python
d = {}
for arg in ('name', 'project_name', 'host', 'id', 'provider'):
    if arg in [att.name for att in klass.ENTITY.__attrs_attrs__]:
        d[arg] = getattr(selected_entity, arg, None)})
return item_collection.instantiate(**d)
```

File: cfme/tests/containers/test_tags.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+
+def _tag_cleanup(test_item, tag):
```
Comment (gshefer) [inline cfme/tests/containers/test_tags.py]: REQUIRED
Please add a short docstring that describes this function

File: cfme/tests/containers/test_tags.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    tags = test_item.get_tags()
+    if tags:
+        result = (
+            not object_tags.category.display_name == tag.category.display_name and
+            not object_tags.display_name == tag.display_name for object_tags in tags
+        )
+        if not result:
```
Comment (gshefer) [inline cfme/tests/containers/test_tags.py]: __REQUIRED__
Is it possible that you forgot `any` or `all` in the `result`? now `result` is generator and always `True`
```

Comment (gshefer) [reply cfme/tests/containers/test_tags.py]: result is always `True`, reason (x for x in <something>) is generator, but probably you wanted to do some operation on it, like `all` or `any`.

**Comment by jawatts** [comment]
@shalomnaim1 Einat wanted the tests completely removed.

**Comment by mshriver** [comment]
@gshefer comments have been addressed, passing tests, merging.

**Pull request merged**